src/server/bootstrap/helmet.ts

```diff
@@ -0,0 +1,21 @@
+import helmet from 'helmet'
+import config from '../config'
+import parseDomain from '../utils/domain'
+
+const cspOnlyReportViolations = config.get('cspOnlyReportViolations')
+const cspReportUri = config.get('cspReportUri')
+const sentryDns = config.get('sentryDns')
+
+export default helmet({
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: ["'self'"],
+      styleSrc: ["'self'", "'unsafe-inline'", 'https://fonts.googleapis.com/'],
+      fontSrc: ["'self'", 'https://fonts.gstatic.com/'],
+      connectSrc: ["'self'", ...(sentryDns ? [parseDomain(sentryDns)] : [])],
```

Comment:
maybe disable the linting here so that sentryDns is on a new line?

Reply:
This is fine